src/main/java/org/myrobotlab/service/DiscordBot.java

```diff
@@ -261,4 +256,81 @@ public void setBotName(String name) {
     this.botName = name;
   }
 
+  @Override
+  public void onImage(ImageData img) {
+    sendImage(img, null, null);
```

Comment:
this means that images by default will always be sent to "general" ... no?  sorry, we need to fix this somehow.